src/style.css

```diff
@@ -1,3 +1,84 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 60px;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+}
+
+ul li {
+  margin: 0;
+  padding-left: 20px;
```

Comment:
Avoid styling using tag names. Instead, style using classes to ensure that styles are not overly broad or accidentally applied to other elements.